src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -106,7 +104,6 @@ public SwerveModule(MotorConfig driveMotorConfig, MotorConfig turningMotorConfig
         turningPIDController.setP(kPTurning);
         simTurningPIDController = new PIDController(turningPIDController.getP(), turningPIDController.getI(), turningPIDController.getD());
         simTurningPIDController.enableContinuousInput(-Math.PI, Math.PI);
-        tunableP = new TunableNumber("Swerve " + ID + " P", kPTurning, turningPIDController::setP);
```

Comment:
Why are we removing this?

Reply:
We don't need a tuner for this, it was just used as an example